Core/Core/View/Base/AlertView.swift

```diff
@@ -397,7 +398,7 @@ public struct AlertView: View {
                                 lineJoin: .round,
                                 miterLimit: 1
                             ))
-                            .foregroundColor(Theme.Colors.textPrimary)
+                            .foregroundColor(Theme.Colors.secondardButtonBorderColor)
```

Comment:
secondar`d`
Should it be `secondary`?

Reply:
It's an overlay/border color, so yes, it should be secondary to match the button styles with other buttons across the app.

Reply:
Typo fixed.